src/Workspaces/Core/Portable/Shared/Extensions/SyntaxGeneratorExtensions.cs

```diff
@@ -612,8 +615,8 @@ private static DeclarationModifiers GetOverrideModifiers(ISymbol symbol)
                     through = generator.CastExpression(interfaceBeingImplemented,
                         through.WithAdditionalAnnotations(Simplifier.Annotation));
                 }
-                else if (!throughMember.IsStatic &&
-                    throughMember is IPropertySymbol throughMemberProperty &&
+                else if (
+                    throughMember is IPropertySymbol { IsStatic: false } throughMemberProperty &&
                     throughMemberProperty.ExplicitInterfaceImplementations.Any())
```

Comment:
Can you use a collection pattern here, e.g. `ExplicitInterfaceImplementations: [var explicitlyImplementedProperty, ..]`? I believe you could remove `throughMemberProperty at that point.

```suggestion
                    throughMember is IPropertySymbol { IsStatic: false, ExplicitInterfaceImplementations: [var explicitlyImplementedProperty, ..] })
```

Reply:
nice :)